crates/noirc_frontend/src/lexer/lexer.rs

```diff
@@ -331,7 +331,7 @@ impl<'a> Lexer<'a> {
     }
 
     fn parse_block_comment(&mut self) -> SpannedTokenResult {
-        let span = Span::new(self.position..self.position + 1);
```

Comment:
This is a change to how we display this error. Previously we'd only show a red squiggly over the first `*` of the comment but now we go from that `*` to the end of the file.

Motivation being that it's more noticeable and fits better with a "hey your comment is covering everything" error.